frontends/web/src/contexts/LocalizationProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react';
+import { useLocalizedPunctuation } from '../hooks/localized';
+import LocalizationContext from './LocalizationContext';
+
+type TProps = {
+    children: ReactNode;
+  }
+
+export const LocalizationProvider = ({ children }: TProps) => {
+  const { decimal, group } = useLocalizedPunctuation('');
+
+  return (
+    <LocalizationContext.Provider value={{ decimal, group }}>
+      {children}
+    </LocalizationContext.Provider>
+  );
+};
```

Comment:
nit: most of the codebase uses lower-case filenames, but there are already a few files in context that contain uppercase letters... maybe start using lowercase here?